src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -144,6 +145,7 @@ public CftQueryService(CamundaService camundaService, CFTTaskMapper cftTaskMappe
             .filter(result -> result.containsKey(TASK_TYPE.value()))
             .map(result -> camundaService.getVariableValue(result.get(TASK_TYPE.value()), String.class))
             .filter(Objects::nonNull)
+            .filter(StringUtils::hasText)
```

Comment:
can filter only once since hasText check for null :) 
```suggestion
            .filter(StringUtils::hasText)
```

Reply:
forgot to remove ..argh....doing it now.